core/src/main/resources/META-INF/mule-module.properties

```diff
@@ -215,7 +215,10 @@ artifact.export.classPackages=org.mule.runtime.core,\
                               javax.net.ssl,\
                               org.apache.commons.io.filefilter,\
                               javax.net,\
-                              org.reactivestreams, \
+                              org.reactivestreams,\
```

Comment:
i will add reactor as a dependency of extension-support to avoid exposing it. It makes no sense to build a facade for it IMO